packages/client/wallets/smart-wallet/src/index.ts

```diff
@@ -11,6 +11,8 @@ export type {
     WalletParams,
 } from "./types/params";
 
+export type { SmartWalletChain } from "./blockchain/chains";
```

Comment:
I wonder if this type should be EVMSmartWalletChain. I think in the future we may benefit from disallowing one to pass "type": "solana-smart-wallet", "defaultChain": "polygon-amoy"

Reply:
makes sense, renamed.